### PR TITLE
Jacklli dev tidb operator deployment private operator

### PR DESCRIPTION
**Pull request by Jacklli**
Jacklli dev tidb operator deployment private operator

### CODE REVIEW

File: session2/chapter1/tidb-operator-deployment-private-env.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+fs.file-max=1000000
+fs.inotify.max_user_watches=1048576
+fs.inotify.max_user_instances=1024
+net.ipv4.conf.all.rp_filter=1
+net.ipv4.neigh.default.gc_thresh1=80000
+net.ipv4.neigh.default.gc_thresh2=90000
+net.ipv4.neigh.default.gc_thresh3=100000
+net.bridge.bridge-nf-call-iptables=1
+net.bridge.bridge-nf-call-arptables=1
+net.bridge.bridge-nf-call-ip6tables=1
```
Comment (tennix) [inline session2/chapter1/tidb-operator-deployment-private-env.md:26]: 这块用表格或者代码块吧，不然会被渲染成一行，没法看

File: session2/chapter1/tidb-operator-local-deployment.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+输出以上信息，则说明 Helm 客户端与服务端都符合要求
+
+## 第二部分： 基于 K8s 部署 TiDB Operator
+
+### 操作步骤如下:
+
+#### 一、通过 helm 安装 TiDB Operator
+创建 TiDB CRD
+```
+    # kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && kubectl get crd tidbclusters.pingcap.com
```
Comment (tennix) [inline session2/chapter1/tidb-operator-local-deployment.md]: 这里 crd 保持跟下面的 operator 版本一致，比如下面是 v1.0.6，那这里就应该是 https://raw.githubusercontent.com/pingcap/tidb-operator/v1.0.6/manifests/crd.yaml

File: session2/chapter1/tidb-operator-deployment-private-operator.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+### 操作步骤如下:
+
+#### 一、通过 helm 安装 TiDB Operator
+创建 TiDB CRD
+```
+    # kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/crd.yaml && kubectl get crd tidbclusters.pingcap.com
```
Comment (tennix) [inline session2/chapter1/tidb-operator-deployment-private-operator.md]: 这里同样要改成跟下面版本一致 https://raw.githubusercontent.com/pingcap/tidb-operator/v1.0.6/manifests/crd.yaml

**Comment by Jacklli** [comment]
close

**Pull request closed without merging**